test/v2-migration-tests/src/test/java/software/amazon/awssdk/v2migrationtests/TestUtils.java

```diff
@@ -41,22 +45,22 @@ public static void assertTwoDirectoriesHaveSameStructure(Path a, Path b) {
         assertLeftHasRight(b, a);
     }
 
+    public static boolean versionAvailable(String sdkVersion) {
+        try {
+            return SdkVersionUtils.checkVersionAvailability(sdkVersion,
+                                                            "apache-client",
+                                                            "netty-nio-client",
+                                                            "dynamodb",
+                                                            "sqs",
+                                                            "s3");
+        } catch (Exception exception) {
+            return false;
```

Comment:
Maybe log a warning or the error here? We might want to have visibility when we expect tests annotated with `@EnabledIf("versionAvailable")` to not be excluded, but they were because of an error here.

Reply:
Sure, updated